docs/en/connector-v2/source/LocalFile.md

```diff
@@ -254,6 +254,72 @@ Specifies Whether to process data using the tag attribute format.
 
 Filter pattern, which used for filtering files.
 
+The filtering format is similar to wildcard matching file names in Linux.
+
+| Wildcard     | Meaning                                                                                                                        | Example                                                                                                                                      |
+|--------------|--------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| *            | Match 0 or more characters                                                                                                     | f* &emsp;&ensp;&emsp; Any file starting with f<br/>b*.txt &emsp; Any file starting with b, any character in the middle, and ending with. txt |
+| []           | Match a single character in parentheses                                                                                        | [abc]* &emsp; A file that starts with any one of the characters a, b, or c                                                                   |
+| ?            | Match any single character                                                                                                     | f?.txt &emsp; Any file starting with 'f' followed by a character and ending with '. txt'                                                     |
+| [!]          | Match any single character not in parentheses                                                                                  | [!abc]* &emsp; Any file that does not start with abc                                                                                         |
+| [a-z]        | Match any single character from a to z                                                                                         | [a-z]* &emsp; Any file starting with a to z                                                                                                  |
+| {a,b,c}/a..z | When separated by commas, it represents individual characters<br/>When separated by two dots, represents continuous characters | {a,b,c}* &emsp; Files starting with any character from abc<br/>{a..Z}* &emsp;&ensp; Files starting with any character from a to z            |
+
+However, it should be noted that unlike Linux wildcard characters, when encountering file suffixes, the middle dot cannot be omitted.
+
+For example, `abc20241022.csv`, the normal Linux wildcard `abc*` is sufficient, but here we need to use `abc*.*` , Pay attention to a point in the middle.
```

Comment:
Please replace this part to link to https://en.wikipedia.org/wiki/Regular_expression. Let user to learn regular itself.